### PR TITLE
Add com.rafaelmardojai.WebfontKitGenerator

### PR DESCRIPTION
**Pull request by rafaelmardojai**
https://github.com/rafaelmardojai/WebfontKitGenerator

### CODE REVIEW

**Comment by rafaelmardojai** [comment]
bot, build com.rafaelmardojai.WebfontKitGenerator

File: com.rafaelmardojai.WebfontKitGenerator.json [hunk 1]
```diff
@@ -0,0 +1,91 @@
+{
+    "app-id":"com.rafaelmardojai.WebfontKitGenerator",
+    "runtime":"org.gnome.Platform",
+    "runtime-version":"3.36",
+    "sdk":"org.gnome.Sdk",
+    "command":"webfontkitgenerator",
+    "finish-args":[
```
Comment (bilelmoussaoui) [inline com.rafaelmardojai.WebfontKitGenerator.json:7]: You probably need `--device=dri`

File: com.rafaelmardojai.WebfontKitGenerator.json [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+                "-Dglade_catalog=disabled"
+            ],
+            "cleanup":[
+                "/include",
+                "/lib/pkgconfig"
+            ],
+            "sources":[
+                {
+                    "type":"git",
+                    "url":"https://source.puri.sm/Librem5/libhandy.git",
```
Comment (bilelmoussaoui) [inline com.rafaelmardojai.WebfontKitGenerator.json:71]: It was migrated to GNOME infra recently
```suggestion
                    "url":"https://gitlab.gnome.org/GNOME/libhandy.git",
```

**Comment by LeandroStanger** [comment]
bot, build com.rafaelmardojai.WebfontKitGenerator

**Comment by rafaelmardojai** [comment]
If there's nothing left, feel free to merge.

**Comment by barthalion** [comment]
LGTM, thanks!

/merge

**Pull request closed without merging**